timemachine/cpp/src/kernels/k_exchange.cu

```diff
@@ -683,10 +683,15 @@ void __global__ k_adjust_sample_idx(
     const int *__restrict__ partitioned_indices,    // [inner_count]
     int *__restrict__ sample_idxs                   // [1]
 ) {
+    int idx = blockIdx.x * blockDim.x + threadIdx.x;
+    // At the moment we only have one sample
```

Comment:
nit: seems like this comment might make more sense (and be less likely to go stale) where the kernel is launched with one thread per block?

Reply:
edit: sorry, I misread this initially. I think this makes sense given the `if (idx > 0)` return condition below.